Description: Update the program version info in the manual pages.
Forwarded: not-yet
Author: Peter Pentchev <roam@ringlet.net>
Bug-Debian: http://bugs.debian.org/744125
Last-Update: 2014-09-14

--- a/spipe/spipe.1
+++ b/spipe/spipe.1
@@ -21,7 +21,7 @@
 .\" LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
 .\" OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF
 .\" SUCH DAMAGE.
-.TH SPIPE 1 "October 10, 2014" "spiped 1.4.2" "spipe README"
+.TH SPIPE 1 "February 22, 2015" "spiped 1.5.0" "spipe README"
 .SH NAME
 spipe \- spiped client utility
 .SH SYNOPSIS
--- a/spiped/spiped.1
+++ b/spiped/spiped.1
@@ -21,7 +21,7 @@
 .\" LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
 .\" OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF
 .\" SUCH DAMAGE.
-.TH SPIPED 1 "October 10, 2014" "spiped 1.4.2" "spiped README"
+.TH SPIPED 1 "February 22, 2015" "spiped 1.5.0" "spiped README"
 .SH NAME
 spiped \- secure pipe daemon
 .SH SYNOPSIS
